dev-lang/ldc2/ldc2-1.24.0-r2.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multilib-build cmake llvm
+
+MY_PV="${PV//_/-}"
+MY_P="ldc-${MY_PV}-src"
+SRC_URI="https://github.com/ldc-developers/ldc/releases/download/v${MY_PV}/${MY_P}.tar.gz"
+S=${WORKDIR}/${MY_P}
+
+DESCRIPTION="LLVM D Compiler"
+HOMEPAGE="https://github.com/ldc-developers/ldc"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+LICENSE="BSD"
+SLOT="$(ver_cut 1-2)/$(ver_cut 3)"
+
+IUSE="static-libs"
+
+# We support LLVM 6.0 through 10.
+RDEPEND="dev-util/ninja
+	|| (
+		sys-devel/llvm:10
+		sys-devel/llvm:11
+	)
+	<sys-devel/llvm-12:=
+	>=app-eselect/eselect-dlang-20140709"
+DEPEND="${RDEPEND}"
+LLVM_MAX_SLOT=10
```

Comment:
The comment on top should match what upstream says they support, i.e. the 10 should be 11. Similarly 12 for the 1.25 ebuild.
Set `LLVM_MAX_SLOT=11` and `12` respectively. IIRC it protects the ebuild against compilation with a too recent LLVM in the future.

Reply:
I modified the dependency versions based on that information about upstream support. But I didn't notice this LLVM_MAX_SLOT variable at all. Those are now updated.
(Although I wonder what I ended up using before changing these)

Reply:
It seems I was actually using LLVM 10 before which makes sense based on what that LLVM_MAX_SLOT variable is described to do. So I hadn't actually tested these at all with newer LLVM version earlier like I thought I had :D

Now I recompiled ldc2 1.26.0-r1 with LLVM 12 and onedrive with it and that seems to work.